daemons/execd/Makefile.am

```diff
@@ -27,7 +28,6 @@ pacemaker_execd_SOURCES	= pacemaker-execd.c 	\
 			  execd_commands.c 	\
 			  execd_alerts.c
 
-if BUILD_REMOTE
```

Comment:
also drop all the C checks for `HAVE_GNUTLS_GNUTLS_H`

Reply:
Fixed.  Note that this removes guards from around members of `pcmk__remote_s` and a couple flag enums, among other spots.  I assume this is not going to be a problem because we won't be supported clusters consisting of v3.0 and v2.x at the same time.